site/content/en/references/kustomize/kustomization/replacements/_index.md

```diff
@@ -0,0 +1,258 @@
+---
+title: "replacements"
+linkTitle: "replacements"
+type: docs
+weight: 18
+description: >
+    Substitute field(s) in N target(s) with a field from a source.
+---
+
+Replacements are used to copy fields from one source into any
+number of specified targets. The `replacements` field in the kustomization file can support either a path to 
+a replacement or an inline replacement. 
+
+`kustomization.yaml`
+```yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+replacements:
+  - path: replacement.yaml
+  - source:
+      kind: Deployment
+      fieldPath: metadata.name
+     targets:
+       - select: 
+           name: my-resource
+```
+
+### Syntax
+
+The full schema of `replacements` is as follows:
+
+```yaml
+replacements:
+- source:
+    group: string
+    version: string
+    kind: string
+    name: string
+    namespace: string
+    fieldPath: string
+    options:
+      delimiter: string
+      index: int
+      create: bool
+  targets: 
+  - select:
+      group: string
+      version: string
+      kind: string
+      name: string
+      namespace: string
+    reject:
+    - group: string
+      version: string
+      kind: string
+      name: string
+      namespace: string
+    fieldPaths: 
+    - string
+    options:
+      delimiter: string
+      index: int
+      create: bool
+```
+
+### Field Descriptions
+
+| Field       | Description |
+| ----------- | ----------- |
+| source [required] | The source of the value |
```

Comment:
Suggest another column `required` with a check for fields where it's required and nothing for optional fields (cut down on text.

 Also always use `code font` for field names and right justify the column

| Field | Required|  |
| ---: | :---: | --- |
| `source` | &#10004; | The source of the value |
| `group` |  | The API group of the value |

Reply:
done